src/crates-io/crate-removal.md

```diff
@@ -1,48 +0,0 @@
-# Crate removal procedure
-
```

Comment:
maybe keep a link to the moved instructions here for people that are used to looking in this place? (not sure if there are any 😅)